vignettes/simulist.Rmd

```diff
@@ -151,37 +169,29 @@ For an overview of how a line list can be simulated with age-stratified (or age-
 
 ## Simulate contacts table
 
-To simulate a contacts table, the `sim_contacts()` function can be used. This requires a contacts distribution (i.e. a distribution describing the variability in number of contacts for different individuals within the population). 
+To simulate a contacts table, the `sim_contacts()` function can be used. This requires the same arguments as `sim_linelist()`, but does not require the onset-to-hospitalisation delay and onset-to-death delays.
 
 ```{r, sim-contacts}
-contact_distribution <- epiparameter::epidist(
-  disease = "COVID-19",
-  epi_dist = "contact_distribution",
-  prob_distribution = "pois",
-  prob_distribution_params = c(l = 5)
-)
-
 contacts <- sim_contacts(
-  R = 1.1,
-  serial_interval = serial_interval,
-  contact_distribution = contact_distribution
-)
+  contact_distribution = contact_distribution,
+  contact_interval = contact_interval,
+  prob_infect = 0.5)
 head(contacts)
 ```
 
 ## Simulate both line list and contacts table
 
 To produce both a line list and a contacts table for the same outbreak, the `sim_linelist()` and `sim_contacts()` cannot be used separately due to the stochastic algorithm, meaning the data in the line list will be discordant with the contacts table. 
 
-In order to simulate a line list and a contacts table of the same outbreak the `sim_outbreak()` function is required. This will simulate a single outbreak and return a line list and a contacts table. The inputs of `sim_outbreak()` are a combination of the inputs required for `sim_linelist()` and `sim_contacts()`.
+In order to simulate a line list and a contacts table of the same outbreak the `sim_outbreak()` function is required. This will simulate a single outbreak and return a line list and a contacts table. The inputs of `sim_outbreak()` are the same as the inputs required for `sim_linelist()`.
 
 ```{r, sim-outbreak}
 outbreak <- sim_outbreak(
-  R = 1.1,
-  serial_interval = serial_interval,
+  contact_distribution = contact_distribution,
+  contact_interval = contact_interval,
+  prob_infect = 0.5,
   onset_to_hosp = onset_to_hosp,
-  onset_to_death = onset_to_death,
-  contact_distribution = contact_distribution
+  onset_to_death = onset_to_death
 )
 head(outbreak$linelist)
 head(outbreak$contacts)
```

Comment:
I thought it might be nice at the end of this section to show a visulisation calling `epicontacts` (e.g. below markdown):

We can also visualise the contact dataset using the `epicontacts` package:
```{r}
library(epicontacts)
contact_object <- make_epicontacts(linelist = outbreak$linelist,
                              contacts = outbreak$contacts, 
                              id = "case_name",
                              directed = TRUE)

plot(contact_object) 
```
But the visualisation wasn't capturing the entries in the linelist. I did a quick check for matching, and found
`sum(is.na(match(outbreak$linelist$case_name,outbreak$contacts$to)))=1`
and
`sum(is.na(match(outbreak$linelist$case_name,outbreak$contacts$from)))=18`

Maybe I've misunderstood, but shouldn't all the `outbreak$contacts$from` entries correspond to a case? (Although this may not explain the error, as not so familiar with `epicontacts` - but it's a package it would be nice to integrate more with!)

Reply:
Thanks for the comment. I've check and it seems the {epicontacts} plots are working as expected. I've decided not to include the {epicontacts} plotting in the `simulist.Rmd` vignette as it is already explained in detail in the [`vis-linelist.Rmd`](https://epiverse-trace.github.io/simulist/articles/vis-linelist.html) and I don't want the Get Started vignette to become too long. 

The `sum(is.na(match(outbreak$linelist$case_name,outbreak$contacts$to)))=1` can be explained as the linelist contains the index case that seeds the outbreak, but this is not included in the `$to` column of the contacts table. 

The mismatch for `outbreak$linelist$case_name` and `outbreak$contacts$from` is likely to cases not transmitting to anyone else so they are not recorded in `$from`. 

Please let me know if there are other inconsistencies to check as best to solve any possible bugs early on in development.